plugins/spu2-x/src/SndOut.h

```diff
@@ -32,7 +32,7 @@ static const int SndOutVolumeShift32 = 16 - SndOutVolumeShift; // shift up, not
 // Samplerate of the SPU2. For accurate playback we need to match this
 // exactly.  Trying to scale samplerates and maintain SPU2's Ts timing accuracy
 // is too problematic. :)
-static const int SampleRate = 48000;
+static int SampleRate = 48000;
```

Comment:
This change means that every translation unit that includes this header will have a separate copy of the variable. Any modification to this variable will only be local to that translation unit and not global.